src/libs/actions/Report.js

```diff
@@ -416,6 +501,17 @@ function updateReportWithNewAction(reportID, reportAction) {
 
     Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, reportActionsToMerge);
 
+    // If chat report receives an action with IOU, update IOU object
+    if (reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU) {
+        const chatReport = lodashGet(allReports, reportID);
+        const iouReportID = lodashGet(chatReport, 'iouReportID');
+        if (iouReportID) {
+            updateIOUReportData(iouReportID, reportID);
+        } else {
+            fetchIOUReportID(reportID).then(iouID => updateIOUReportData(iouID, reportID));
+        }
```

Comment:
Group chat reports will meet the `reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.IOU` condition, but we should exclude them, as they cannot have an associated IOUReport.

Reply:
I'm now checking length of participants and don't call the functions for group chats.